### ISSUE TITLE
Android - Search results Screen - The Bold Text at the Top of Each Search Result Item Should Be a Heading

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#25]
The bold text at the top of each search result should be a heading so the user can skip from one item to the next.

**Comment by KyleLivingstonDOL** [comment]
It looks like this is done. UAT will begin on June 10.

**Comment by jfc3-dol** [comment]
TalkBack is reading it as a link but not a heading. Not sure if it can do both. Do you have any knowledge on whether an item can be a heading and a link at the same time from a programming standpoint?

**Comment by LeHaine** [comment]
The anchor tag (the link) is currently wrapped around an h2 tag. There isn't a way to make an item a link and a heading under a single HTML element. They have to wrap around each other such as they are doing now.

**Comment by jfc3-dol** [comment]
Whenever I create HTML links that are a heading I wrap the heading around the link, so it reads both.

See the below example code of what I'm talking about:

<h1><a href="default.html">john f croston iii</a></h1>

Coding it this way will be read as "john f croston iii heading level one link" at least for VoiceOver and something similar for other screen readers.

**Comment by LeHaine** [comment]
I noticed my previous comment had the two elements backwards. Sorry about that. The way you mentioned though is exactly how it is now except with an h2.

**Comment by jfc3-dol** [comment]
Okay, maybe it's the way TalkBack reads things. I'm more used to using JAWS and VoiceOver.

Will check again with the next build.

**Comment by jfc3-dol** [comment]
TalkBack is now reading the larger bold text at the top of the screen as a heading level one for the Step screens. The same applies for the few Topics screens I looked at too.

So I'm going to close this issue.
